### PR TITLE
Create dashboard entry for marc21

### PR DESCRIPTION
**Pull request by philippgualdi**
Add a dashboard entry for marc21 records to user dashboard

**DEMO**
image

### CODE REVIEW

File: invenio_records_marc21/templates/invenio_records_marc21/uploads.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+<div class="ui divider hidden"></div>
+{%- endblock frontpage_form %}
+
+    <div id="invenio-search-marc21-config" data-invenio-search-config='{{ search_app_marc21_config() | tojson }}'></div>
+</div>
+
+
+
+{%- endblock page_body %}
+
```
Comment (utnapischtim) [inline invenio_records_marc21/templates/invenio_records_marc21/uploads.html:54]: could you please add a end of line at the end of the file

File: invenio_records_marc21/templates/invenio_records_marc21/uploads.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+</div>
+
+
+<div class="ui divider hidden"></div>
+{%- endblock frontpage_form %}
+
+    <div id="invenio-search-marc21-config" data-invenio-search-config='{{ search_app_marc21_config() | tojson }}'></div>
+</div>
+
+
```
Comment (utnapischtim) [inline invenio_records_marc21/templates/invenio_records_marc21/uploads.html:51]: minor: remove the not necessary empty lines

File: invenio_records_marc21/ui/theme/__init__.py [hunk 3]
```diff
@@ -26,10 +26,23 @@ def init_theme_views(blueprint, app):
     routes = app.config.get("INVENIO_MARC21_UI_THEME_ENDPOINTS")
 
     blueprint.add_url_rule(routes["index"], view_func=index)
+
+    blueprint.add_url_rule(routes["uploads-marc21"], view_func=uploads_marc21)
     blueprint.add_url_rule(routes["record-search"], view_func=search)
     blueprint.add_url_rule(routes["deposit-create"], view_func=deposit_create)
     blueprint.add_url_rule(routes["deposit-edit"], view_func=deposit_edit)
 
+    @blueprint.before_app_first_request
```
Comment (utnapischtim) [inline invenio_records_marc21/ui/theme/__init__.py:35]: Add a TODO to fix this after https://github.com/inveniosoftware/invenio-base/issues/171 has been solved

**Review by utnapischtim — CHANGES_REQUESTED** [review]

File: invenio_records_marc21/ui/theme/__init__.py (hunk 3 above)
Comment (philippgualdi) [reply invenio_records_marc21/ui/theme/__init__.py:36]: A fix me note was added

**Review by utnapischtim — APPROVED** [review]

**Pull request merged**